docs/source/ultimate.rst

```diff
@@ -59,6 +59,70 @@ Nous garderons la partie privée de vos données (nom, coordonnées, etc.) dans
+
+I dati che verranno condivisi con il pubblico non saranno identificati dal tuo nome ma solo grazie ad un codice numerico, in modo che nessuno possa identificarti o sapere quali dati sono tuoi. Inoltre, al meglio delle nostre conoscenze, non condivideremo altre informazioni che potrebbero essere usate da qualcuno che ti conosce per risalire a quali sono i tuoi dati.
+
+Se tu dovessi cambiare idea e volessi ritirare il tuo consenso alla partecipazione a questo studio (per farlo, chiama <Nome Responsabile> al numero <Numero Telefono>), non raccoglieremo dati aggiuntivi su di te. Se tu dovessi ritirare il tuo consenso prima che i dati siano depositati nella banca dati, questi saranno eliminati. **Tuttavia, dati e risultati eventualmente già condivisi con altri investigatori o con il grande pubblico non possono essere distrutti, rimossi o ritirati.**
+
+Accettando di partecipare, farai un dono gratuito e generoso alla ricerca scientifica, potenzialmente utile ad altri. È possibile che alcune delle ricerche condotte usando i tuoi dati portino allo sviluppo di nuovi metodi per studiare il cervello, nuovi strumenti diagnostici, nuovi farmaci o altri prodotti commerciali. Se questo dovesse accadere, non riceverai alcuna parte dei profitti generati da tali prodotti sui quali non potrai avanzare alcun diritto di proprietà.
+
+Per quanto ci è possibile sapere, i dati che verranno condivisi con il grande pubblico non conterranno alcuna informazione che sia direttamente ricollegabile a te. I dati che verranno condivisi non saranno identificati dal tuo nome ma solo grazie ad un codice numerico in modo tale da garantirne l'anonimato. Inoltre, non includeremo dati che pensiamo possano permettere alle persone che ti conoscono di indovinare quali dati siano tuoi, come ad esempio le caratteristiche del tuo viso o la data in cui hai partecipato. Se dovessimo scrivere un articolo o report su questo studio, o se dovessimo condividere con altri i dati raccolti durante questo studio, lo faremo in modo da non compromettere il tuo anonimato. Tuttavia, utilizzando altri dati connessi al tuo nome (ad esempio delle immagini del tuo cervello ottenute dalla tua cartella clinica), potrebbe essere possibile ricollegarti alle immagini o altri dati del nostro database. Inoltre, una violazione della sicurezza od un attacco informatico potrebbero permettere a qualcuno di ricollegarti ai dati anonimizzati. Queste rischio è minimo, dato che i tuoi dati saranno conservati in un database sicuro e le informazioni sulla tua identità conservate separatamente, collegate solo da un codice.
+
+Conserveremo i tuoi dati sensibili (nome e cognome, informazioni per contattarti etc...) in un luogo sicuro per <x> anni.  In questo modo, se uno dei ricercatori aventi accesso ai tuoi dati dovesse scoprire qualcosa di utile per te dal punto di vista diagnostico, saremmo in grado di contattarti. Passato questo intervallo di tempo, distruggeremo queste informazioni per proteggere la tua privacy.
```

Comment:
Conserveremo i tuoi dati sensibili **(nome e cognome, informazioni per contattarti, ecc.)** in un luogo sicuro per \<x\> anni.  In questo modo, se uno dei ricercatori aventi accesso ai tuoi dati dovesse scoprire qualcosa di utile per te dal punto di vista diagnostico, **saremo** in grado di contattarti. Passato questo intervallo di tempo, distruggeremo queste informazioni per proteggere la tua privacy.